nmmo/io/action.py

```diff
@@ -275,6 +275,11 @@ def call(realm, entity, style, target):
     if style.freeze and dmg > 0:
       target.status.freeze.update(config.COMBAT_FREEZE_TIME)
 
+    # record the combat tick for both player entities
+    for ent in [entity, target]:
+      if ent.is_player:
+        ent.latest_combat_tick = realm.tick + 1 # because the tick is about to increment
+
```

Comment:
this won't catch cases where an NPC attacks the player. we should be setting the tick in enity.receiveDamage and entity.giveDamage